donate/templates/fragments/donate_form.html

```diff
@@ -19,30 +19,10 @@
             <form method="GET" id="donate-form--single">
                 <input type="hidden" name="currency" value="usd">
 
-                <div class="donate-form__fields">
-                    <div class="donation-amount">
-                        <input type="radio" class="donation-amount__radio" name="amount" value="50" id="one-time-amount-1" autocomplete="off">
-                        <label for="one-time-amount-1" class="donation-amount__label">$50</label>
-                    </div>
-                    <div class="donation-amount">
-                        <input type="radio" class="donation-amount__radio" name="amount" value="25" id="one-time-amount-2" autocomplete="off">
-                        <label for="one-time-amount-2" class="donation-amount__label">$25</label>
-                    </div>
-                    <div class="donation-amount">
-                        <input type="radio" class="donation-amount__radio" name="amount" value="10" id="one-time-amount-3" autocomplete="off">
-                        <label for="one-time-amount-3" class="donation-amount__label">$10</label>
-                    </div>
-                    <div class="donation-amount">
-                        <input type="radio" class="donation-amount__radio" name="amount" value="3" id="one-time-amount-4" autocomplete="off" checked>
-                        <label for="one-time-amount-4" class="donation-amount__label">$3</label>
-                    </div>
-                    <div class="donation-amount donation-amount--two-col donation-amount--other">
-                        <input type="radio" class="donation-amount__radio" name="amount" value="other" id="one-time-amount-other" autocomplete="off">
-                        <label for="one-time-amount-other" class="donation-amount__label">$</label>
-                        <input type="text" class="donation-amount__input" id="one-time-amount-other-input" placeholder="{% trans 'Other amount' %}">
-                    </div>
-                    <input type="hidden" class="one-time-amount__total">
+                <div class="donate-form__fields" id="js-donate-form-single">
+                    {# Filled with js #}
```

Comment:
Do you think we should render the initial set of options in the back end, and then only use JS for subsequent changes? I am not sure it matters a great deal in the sense that nothing will work anyway if JS is disabled, bit it would result in a faster render for initial page load.

Reply:
I ended up adding a min-height to the form (assuming the form has 4 radio options plus other field) to ensure there was no initial jump on load. But it might make sense to render this via BE if it's easy to do. Like you say, would increase the initial page load!